__tests__/api.test.js

```diff
@@ -106,3 +106,48 @@ describe("/api/articles/:article_id", () => {
       });
   });
 });
+
+describe.only("/api/articles/:article_id/comments", () => {
+  test("GET:200. Sends an array of comments belonging to a single article to the client", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.comments).toEqual(expect.any(Array));
+        body.comments.forEach((comment) => {
```

Comment:
You will still want to assert the length of the array here otherwise the forEach will not run

Reply:
Done